product_multi_barcode/__manifest__.py

```diff
@@ -15,11 +15,12 @@
     "ADHOC SA, "
     "Odoo Community Association (OCA)",
     "category": "Product Management",
-    "depends": ["product"],
+    "depends": ["product", "stock"],
```

Comment:
IMHO adding a new dependency should be avoided.

You can add a new module instead.

Reply:
ok